numba/targets/numbers.py

```diff
@@ -79,60 +79,98 @@ def int_divmod(context, builder, x, y):
     *p_xmody = xmody;
     """
     assert x.type == y.type
-    xdivy = builder.sdiv(x, y)
-    xmody = builder.srem(x, y)  # Intel has divmod instruction
 
-    ZERO = Constant.null(y.type)
-    ONE = Constant.int(y.type, 1)
+    ZERO = y.type(0)
+    ONE = y.type(1)
 
-    y_xor_xmody_ltz = builder.icmp(lc.ICMP_SLT, builder.xor(y, xmody), ZERO)
-    xmody_istrue = builder.icmp(lc.ICMP_NE, xmody, ZERO)
-    cond = builder.and_(xmody_istrue, y_xor_xmody_ltz)
+    # NOTE: On x86 at least, dividing the smallest representable integer
+    # by -1 causes a SIFGPE (division overflow), causing the process
```

Comment:
It may be a dept. of engineering colloquialism :-) Adding an example to clarify would be great, thanks. Seems like "minimum value" is used too cf. `limits.h`.